cmd/preview/list.go

```diff
@@ -73,15 +88,66 @@ func executeListPreviews(ctx context.Context, opts ListFlags) error {
 		return fmt.Errorf("failed to get preview environments: %s", err)
 	}
 	w := tabwriter.NewWriter(os.Stdout, 1, 1, 2, ' ', 0)
-	fmt.Fprintf(w, "Name\tScope\tSleeping\tLabels\n")
-	for _, preview := range previewList {
-		previewLabels := "-"
-		if len(preview.PreviewLabels) > 0 {
-			previewLabels = strings.Join(preview.PreviewLabels, ", ")
+	switch opts.output {
+	case "json":
+		var previewSlice []PreviewJson
+		for _, preview := range previewList {
+			previewLabels := "-"
+			if len(preview.PreviewLabels) > 0 {
+				previewLabels = strings.Join(preview.PreviewLabels, ", ")
+			}
+			previewJson := PreviewJson{
+				Name:     preview.ID,
+				Scope:    preview.Scope,
+				Sleeping: preview.Sleeping,
+				Labels:   previewLabels,
+			}
+			previewSlice = append(previewSlice, previewJson)
+		}
+		bytes, err := json.MarshalIndent(previewSlice, "", " ")
+		if err != nil {
+			return err
+		}
+		oktetoLog.Println(string(bytes))
+	case "yaml":
+		var previewSlice []PreviewJson
```

Comment:
```suggestion
		var previewSlice []PreviewOutput
```
since it is used for control the output in both cases

Reply:
Resolved